docs/changelog.rst

```diff
@@ -6,6 +6,11 @@ All notable changes to vimiv are documented in this file.
 v0.10.0 (unreleased)
 --------------------
 
+Fixed:
+^^^^^^
+
+* The image header check for SVG was faulty and returned true for all XML files.
```

Comment:
This sounds a bit harsh, but feel free to keep :blush: Would prefer something along:
Improved the SVG image header check to no longer return true for all XML files.

Reply:
I have improved the wording